lib/shopify_api/session.rb

```diff
@@ -180,6 +181,15 @@ def state
 
     private
 
+    def valid_access_scopes?
```

Comment:
Will this ever be false?

Reply:
No because a bad input will cause `ShopifyAPI::ApiAccess` to raise an error

Reply:
Do you think we should even have the validity check for access scopes then?

Reply:
I think that's one thing I was uncertain about as well. Either `access_scopes=` will throw an error, or it will correctly be `ShopifyAPI::ApiAccess`. Should this determine whether the entire session is valid?

Reply:
I think given that `ShopifyAPI::ApiAccess` rejects it if you try to pass in a bad input, we won't need a `valid?` check. `valid_access_scopes?` currently will always resolve to true.